### PR TITLE
Timestamp can be fed to image in initialzation

### PR DESCRIPTION
**Pull request by EStorvik**
Included the possibility for Image to read timestamps. There is no explicit check that they are in datetime.datetime format. Do you think that should be added? I am honestly not to familiar with this.

### CODE REVIEW

File: src/daria/image/image.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class Image:
             origo (np.ndarray): physical coordinates of the lower left corner, i.e.,
                 of the (img.shape[0],0) pixel
             color_space (str): Color space (RGB/BGR/RED/GREEN/BLUE/GRAY)
+            timestamp (datetime.datetime): timestamp of the image in datetime format.
```
Comment (jwboth) [inline src/daria/image/image.py:46]:
```suggestion
            timestamp (datetime.datetime): timestamp of the image in (default) datetime format.
```

File: src/daria/image/image.py [hunk 2]
```diff
@@ -139,7 +140,7 @@ def __init__(
             self.metadata["color_space"] = "BGR"
             self.metadata["original_dtype"] = self.img.dtype
 
-            if get_timestamp:
+            if self.metadata["timestamp"] is None:
                 pil_img = PIL_Image.open(Path(img))
 
                 # Read exif metadata
```
Comment (jwboth) [inline src/daria/image/image.py:146]:
```suggestion
                # Read timestamp from exif metadata if existent
```

**Review by jwboth — APPROVED** [review]

**Comment by jwboth** [comment]
I think it looks good for now. Let's see if adaptions are needed when we start using this. My recommendation for these type of updates is to simulateneously use the functionality in a use case to see whether the final solution is working out. This leads to less iterations. This is so short, that I would hope that everything works out.

**Comment by EStorvik** [comment]
I agree, but I didn't bother writing the code for transcribing the image names to datetime as Benyamine told us that he already has a plan for it. I think/hope that this will not affect any current code, and we will quickly see if everything works out when Benyamine tries it out. You can merge if you think it is ok!

**Pull request merged**